gazelle/bzl/BUILD

```diff
@@ -51,3 +51,25 @@ gazelle(
     name = "gazelle",
     gazelle = ":gazelle-skylib",
 )
+
+# The files needed for distribution
```

Comment:
Would adding `allow_empty` to the `glob` be an alternative? It wouldnt' be error-prone in this context.

Reply:
According to the docs, `allow_empty` defaults to `True`, but I've made it explicit.